_security/configuration/disable.md

```diff
@@ -43,22 +60,62 @@ If you disable the Security plugin in `opensearch.yml` (or delete the plugin ent
    COPY --chown=opensearch-dashboards:opensearch-dashboards opensearch_dashboards.yml /usr/share/opensearch-dashboards/config/
    ```
 
-   In this case, `opensearch_dashboards.yml` is a "vanilla" version of the file with no entries for the Security plugin. It might look like this:
+1. To build the new Docker image, run the following command:
+
+   ```bash
+   docker build --tag=opensearch-dashboards-no-security .
+   ```
+
+1. In `docker-compose.yml`, change `opensearchproject/opensearch-dashboards:{{site.opensearch_dashboards_version}}` to `opensearch-dashboards-no-security`.
+1. Change `OPENSEARCH_HOSTS` or `opensearch.hosts` to `http://` rather than `https://`.
+1. Enter `docker-compose up`.
+
+### Tarball 
+
+1. Navigate to the `/bin` directory in your OpenSearch Dashboards installation folder and stop the running OpenSearch Dashboards instance by pressing `Ctrl + C`.
+
+1. Run the following command to uninstall the Security plugin. 
+
+   ```bash
+   ./bin/opensearch-dashboards-plugin remove securityDashboards
+   ```
+
+1. Remove all Security plugin related configuration from the opensearch_dashbaords.yml file or use the following example file. 
 
    ```yml
    ---
    server.name: opensearch-dashboards
    server.host: "0.0.0.0"
    opensearch.hosts: http://localhost:9200
    ```
+1. Start OpenSearch Dashboards.
+   ```bash
+   ./bin/opensearch-dashboards
+   ```
+   
+### RPM and Debian 
 
+1. Stop the running instance of OpenSearch Dashboards with the following command: 
 
-1. To build the new Docker image, run the following command:
+   ```bash
+   sudo systemctl stop opensearch-dashboards
+   ```
+
+1. Navigate to the OpenSearch Dashboards folder `/usr/share/opensearch-dashboards` and run the following command to uninstall the Security plugin. 
 
    ```bash
-   docker build --tag=opensearch-dashboards-no-security .
+   ./bin/opensearch-dashboards-plugin remove securityDashboards
    ```
 
-1. In `docker-compose.yml`, change `opensearchproject/opensearch-dashboards:{{site.opensearch_dashboards_version}}` to `opensearch-dashboards-no-security`.
-1. Change `OPENSEARCH_HOSTS` or `opensearch.hosts` to `http://` rather than `https://`.
-1. Enter `docker-compose up`.
+1. Remove all Security plugin related configuration from the opensearch_dashbaords.yml file, or use the example file and place the file in the `/etc/opensearch_dashboards` folder.
```

Comment:
"Remove all Security plugin related configuration" needs a little revision for clarity.